platform/notifications/core/platform-core-notificationcommands/src/main/java/org/codice/ddf/activities/commands/SendCommand.java

```diff
@@ -19,32 +19,37 @@
 import java.util.Map;
 import java.util.UUID;
 
-import org.apache.felix.gogo.commands.Argument;
-import org.apache.felix.gogo.commands.Command;
-import org.apache.karaf.shell.console.OsgiCommandSupport;
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
 import org.codice.ddf.activities.ActivityEvent;
 import org.codice.ddf.activities.ActivityEvent.ActivityStatus;
+import org.osgi.framework.BundleContext;
 import org.osgi.framework.ServiceReference;
 import org.osgi.service.event.Event;
 import org.osgi.service.event.EventAdmin;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Service
 @Command(scope = "activities", name = "send", description = "Send activities.")
-public class SendCommand extends OsgiCommandSupport {
+public class SendCommand implements Action {
     public static final String SERVICE_PID = "org.osgi.service.event.EventAdmin";
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SendCommand.class);
 
     private static final int UNKNOWN_PROGRESS = -1;
 
-    @Argument(name = "User ID", description = "User ID to send notifications to. ", index = 0,
-            multiValued = false, required = true)
-
+    @Argument(name = "User ID", description = "User ID to send notifications to. ", index = 0, multiValued = false, required = true)
     String userId = null;
 
+    @Reference
+    BundleContext bundleContext;
```

Comment:
I think @josephthweatt might have already looked into this, but why not just use
```java
@Reference 
EventAdmin eventAdmin;
```

instead of getting the `eventAdmin` from the `bundleContext`? Same with the other commands in this PR.